index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Projeto</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header id="cabecalho">
+    <h1 id="titulo">Projeto Trybe!</h1>
+  </header>
+  <div class="test">
+    <nav id="navegacao">
+      <ul>
+        <li><a href="https://fedolfo.github.io/" target="_blank" style="text-decoration: none;">Portfólio(open in new window)</a></li>
+        <li><a href="#contato" style="text-decoration: none;">Contato</a></li>
+        <li><a href="#trybe" style="text-decoration: none;">Trybe</a></li>
+      </ul>
+    </nav>
+  </div> 
+  <main>
+
+    <img id="minha_foto"  src="/Filipe_candido.jpg" alt="Foto de uma pessoa chamado Filipe" width="250px">
+    <ol id="licoes_aprendidas">
+      <h2><b>Lições Aprendidas:</b></h2>
+      <li>Unix</li>
+      <li>Bash</li>
+      <li>Git</li>
+      <li>GitHub</li>
+      <li>HTML5</li>
+      <li>CSS</li>
+      <li>HTML Semântico</li>
+      <li>Estilização de Página</li>
+      <li>Acessibilidade</li>
+      <li>Tabelas HTML</li> 
+    </ol>
+    <ul id="licoes_a_aprender">
+      <h2><b> Lições a Aprender: </b></h2>
+      <li>JavaScript</li>
+      <li>Lógica de Programação</li>
+      <li>Forms</li>
+      <li>Flexbox</li>
+      <li>Responsivo</li>
+      <li>ES6</li>
+      <li>Testes Unitários</li>
+      <li>Higher Order Funcions do JavaScript ES6</li>
+      <li>JavaScript Assíncrono é Promisses</li>
+      <li>Testes automatizados com Jest</li>
+    </ul>
+        <table>
+          <caption><b>INFORMAÇÕES</b></caption>
+          <thead>
+            <th>Projeto Trybe: Todo conteúdo!</th>
+          </thead>
+          <tbody>
+          <tr>
+            <td>Lições que Aprendi</td>
+            <td>Lições que vou aprender</td>
+          </tr>
+          <tr>
+            <td>Um Artigo</td>
+            <td>Manias que eu tenho</td>
+          </tr>
+          <tr>
+            <td>Contato</td>
+            <td>Tryber</td>
+          </tr>
+          </tbody>
+        </table>
```

Comment:
Filipe, preferimos deixar a indentação padrão, ou seja, essa table dever estar debaixo da ul.
```suggestion
     <table>
       <caption><b>INFORMAÇÕES</b></caption>
        <thead>
           <th>Projeto Trybe: Todo conteúdo!</th>
       </thead>
       <tbody>
        <tr>
          <td>Lições que Aprendi</td>
          <td>Lições que vou aprender</td>
        </tr>
         <tr>
       <td>Um Artigo</td>
        <td>Manias que eu tenho</td>
       </tr>
          <tr>
            <td>Contato</td>
            <td>Tryber</td>
          </tr>
          </tbody>
        </table>
```